### PR TITLE
Quizmaster

### PR DESCRIPTION
**Pull request by FuroYT**

### CODE REVIEW

**Comment by Ultradragon005** [comment]
holy shid

**Comment by FuroYT** [comment]
finally github isnt shitting itself anymore

**Comment by FuroYT** [comment]
tier 2 progress is for tomorrow ;)

**Comment by spuhnj** [comment]
yippeeee

**Comment by 0xDrMoe** [comment]
@FuroYT ETA? If not ready, this will be in next dev

**Comment by FuroYT** [comment]
I will try to finish it today and tmr

**Comment by FuroYT** [comment]
But before that I'll need to fix conflicts (thanks GitHub)

File: Modules/OptionHolder.cs
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:2980]: Extra option

File: Modules/OptionHolder.cs
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:3152]: ... Why was it deleted?

File: Modules/Utils.cs
Comment (Tommy-XL) [inline Modules/Utils.cs:1958]: Same here

File: Patches/ChatCommandPatch.cs
Comment (Tommy-XL) [inline Patches/ChatCommandPatch.cs:2233]: Why not move this to `Quizmaster.cs`

File: Modules/OptionHolder.cs
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:3473]: Extra options

File: Patches/MeetingHudPatch.cs
Comment (Tommy-XL) [inline Patches/MeetingHudPatch.cs:1520]: Isn't it better to move `Quizmaster.OnMeetingEnd();` after `if (AmongUsClient.Instance.AmHost)`

File: Patches/PlayerControlPatch.cs
Comment (Tommy-XL) [inline Patches/PlayerControlPatch.cs:1347]: Same here

File: Modules/ExtendedPlayerControl.cs
Comment (Tommy-XL) [inline Modules/ExtendedPlayerControl.cs:747]: Build Error?

**Review by Tommy-XL — CHANGES_REQUESTED** [review]
So

**Comment by FuroYT** [comment]
> So

So um GitHub did shit I think I just pulled the upstream

File: Modules/Utils.cs
Comment (FuroYT) [reply Modules/Utils.cs:1958]: this was intentional so it shows the tag even if you haven't the mod running

**Comment by FuroYT** [comment]
So i've finished i will ask for code review to check if i ain't broke nuthin

**Comment by FuroYT** [comment]
> @FuroYT ETA? If not ready, this will be in next dev

It is ready, now just waiting for review from Tommy for him to check

**Comment by Tommy-XL** [comment]
> It is ready, now just waiting for review from Tommy for him to check

I'll check the code later

**Comment by FuroYT** [comment]
> > It is ready, now just waiting for review from Tommy for him to check
> 
> I'll check the code later

np :)

File: Modules/CustomRolesHelper.cs
Comment (Gurge44) [inline Modules/CustomRolesHelper.cs:263]: This function is used to make desync impostor roles (neutrals and crewmates with impostor basis) and you don't need to add normal impostor roles to this

**Comment by Tommy-XL** [comment]
Need fix json again :hehe_sob:

**Comment by Tommy-XL** [comment]
And roleColor.json

**Comment by FuroYT** [comment]
> Need fix json again :hehe_sob: 

Can you just get the latest changes from the json and put it cuz I don't know how to fix this one out

**Comment by Tommy-XL** [comment]
> Can you just get the latest changes from the json and put it cuz I don't know how to fix this one out

You can undo all changes to .json and add from via github
image

**Comment by FuroYT** [comment]
> You can undo all changes to .json and add from via github
> 
> image
> 
> 

I'll do it when I get home

**Comment by 0xDrMoe** [comment]
Please resolve this. Let's get it into this dev version.

**Comment by FuroYT** [comment]
> Please resolve this. Let's get it into this dev version.

I’ll do it as quick as I can with school and personal life :]

**Comment by FuroYT** [comment]
@Tommy-XL Finished the json void problem :]

**Comment by 0xDrMoe** [comment]
Is this ready then?

File: Modules/OptionHolder.cs
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:3276]: This should be hidden for H&S

File: Modules/OptionHolder.cs
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:3331]: Same

File: Modules/OptionHolder.cs
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:3549]: Remove .`SetGameMode(CustomGameMode.Standard);`

File: Modules/OptionHolder.cs
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:3531]: Same

**Review by Tommy-XL — APPROVED** [review]

**Pull request merged**
